core/src/main/java/io/github/tcdl/msb/impl/ResponderServerImpl.java

```diff
@@ -47,7 +47,7 @@ static ResponderServerImpl create(String namespace,  MessageTemplate messageTemp
     public ResponderServer listen() {
         ChannelManager channelManager = msbContext.getChannelManager();
 
-        channelManager.subscribe(namespace,
+        boolean isAlreadySubscribed = channelManager.subscribe(namespace,
```

Comment:
>This solution may lead to the conflict between DefaultChannelMonitorAggregator and ResponderServer

Agree,we can add the same flag for DefaultChannelMonitorAggregator to avoid situation when DefaultChannelMonitorAggregator and ResponderServer use the same topic.